### ISSUE TITLE
Incorrect Symbol for Ottava Bassa 8vb

### ISSUE THREAD

**Issue by Pinary-Pi** [musescore/MuseScore#24964]
### Issue type

Engraving bug (incorrect score rendering)

### Description with steps to reproduce

1. Insert an ottava bassa line anywhere in your score.
2. De-select the "Show numbers only" checkbox under properties and style.
 The text on the ottava bassa line will be "8va" instead of the expected "8vb".

### Supporting files, videos and screenshots

![image](https://github.com/user-attachments/assets/01fea61c-7f1e-4bc7-b6ea-5c687b5c4bae)

### What is the latest version of MuseScore Studio where this issue is present?

OS: Windows 11 Version 2009 or later, Arch.: x86_64, MuseScore Studio version (64-bit): 4.4.2-242570931, revision: github-musescore-musescore-3130f97

### Regression

Yes, this used to work in MuseScore 3.x and now is broken

### Operating system

Windows 11

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by Pinary-Pi** [comment]
![image](https://github.com/user-attachments/assets/9d4be8d8-0c2d-4bc9-bade-1017d6489835)

Thought that this picture could be helpful too. It shows that it is using the "ottavaBassa" symbol. 

Now, I am aware that 8va does mean an octave above or below depending on the context, but 8vb is soo much clearer and in practice I've personally never seen an 8va symbol mean play an octave lower on scores I've used. There should at least be the option to change it.

**Comment by SagarChandra07** [comment]
This might help:This file might fix it

https://mega.co.nz/#!qq4nATTK!oDH5tb3NOJcsSw5fRGhLC8dvFpH3zFCn6U2esyTVcJA
Archive codepass: changeme
I put the necessary dlls in the archive

**Comment by Dima-S-Jr** [comment]
@Pinary-Pi, watch [this comment](https://github.com/musescore/MuseScore/issues/20465#issuecomment-1855146611), and that's it, I think your issue is closed. Your issue is a duplicate.
I advise you to refer to the [SMuFL](https://w3c.github.io/smufl/latest/tables/octaves.html) where you can find out for yourself what is not really a bug.

**Comment by Jojo-Schmitz** [comment]
This isn't incorrect at all, the va is the Italion version of th, 8va == 8th
And there is an option already to chnage to 8vb, if you really need it

**Comment by FrancRos31** [comment]
Why 8va bassa is written 8vb? 8vab would be the right abbreviation.

**Comment by Pinary-Pi** [comment]
> @Pinary-Pi, watch [this comment](https://github.com/musescore/MuseScore/issues/20465#issuecomment-1855146611), and that's it, I think your issue is closed. Your issue is a duplicate.
> I advise you to refer to the [SMuFL](https://w3c.github.io/smufl/latest/tables/octaves.html) where you can find out for yourself what is not really a bug.

Oh thank you! I did not know where this was, so I appreciate the link to the list of ottava options. I also didn't realize this was a duplicate. I looked but I must've missed the other one, sorry about that.

**Comment by Pinary-Pi** [comment]
> This isn't incorrect at all, the va is the Italian version of th, 8va == 8th and whether it is bassa or alta is seen by whether it is below or above the staff.
> And there is an option already to change to 8vb, if you really need it.

Italian doesn't really have a version of 'th' in the same way English does, but yes, I do know 8va = 8th and that it's technically not incorrect so I should've used a better word, sorry. I didn't realize there was an option to change it, but I believe I figured it out now. Thanks for letting me know! I think it should be at least easier to find it, however. Maybe in the form of a check box or something else? 

I'll close this issue though because my issue seems to have been solved.
